MovieApp/MovieApp/App/Features/Movies/ViewControllers/DetailsViewController/DetailsViewController.swift

```diff
@@ -128,3 +129,18 @@ class DetailsViewController: UIViewController {
     }
 
 }
+
+extension DetailsViewController: UIGestureRecognizerDelegate {
+
+    func gestureRecognizer(
+        _ gestureRecognizer: UIGestureRecognizer,
+        shouldBeRequiredToFailBy otherGestureRecognizer: UIGestureRecognizer
+    ) -> Bool {
+        return true
+    }
+
+    private func setGestureRecognizerDelegate() {
```

Comment:
this func is not needed, you could just copy line 143 to line 41 or make it a part of styleViews